### DISCUSSION TITLE
Truncating labels leads to incorrect grouping

### DISCUSSION THREAD

**Discussion by nc** [observablehq/plot#1753] [Q&A]
<img width="441" alt="image" src="https://github.com/observablehq/plot/assets/24895/1adbd172-3b94-4162-80fa-b36d0a79e7d3">

Here three separate rows are grouped in the first bar as I tried truncating the label which leads to the items being the same. Is there another way to achieve this without changing the underlying data?

**Comment by mbostock** [comment]
Can you please share some code that demonstrates the problem? If you’re truncating the group key it’s probably unavoidable that the groups get combined. Perhaps you instead need to use the tickFormat option on the y scale to truncate the axis, or use the axisY mark with the textOverflow and lineWidth options. Either way please share some code (ideally as an Observable notebook) so we can see and help.
